src/AgentMulder.ReSharper.Plugin/Navigation/GotoInstantiationNavigationProvider.cs

```diff
@@ -24,9 +27,26 @@ namespace AgentMulder.ReSharper.Plugin.Navigation
     [ContextNavigationProvider]
     public class GotoInstantiationNavigationProvider : INavigateFromHereProvider
     {
+        [Import(typeof(INavigationProvider))]
+        private INavigationProvider navigationProvider;
```

Comment:
There is a possibility to have more providers, right? Maybe this should import a collection?